experiments/snippets/autoLoadMore/README.md

```diff
@@ -0,0 +1,6 @@
+# autoLoadMore.js
+This inline javascript (NOT snippet) used to load more rows into a model when the "Load More" button is scrolled into the window.  This creates a scroll and load experience like salesforce.  There must be enough rows in the model to allow for a scroll though - I'm sure other trigger events could be provided, but this is based on too many to load at once in a performance friendly experience, and then load more when scrolling long enough.  We tend to use this with the "Max # of records (Limit)" field set between 50-100, but it will dependend on what is being queried and when you see a performance degredation from overloading.  It will require modification of the model and field names to match yours.
```

Comment:
Some copy/grammar edits.

```suggestion
This inline JavaScript (NOT snippet) used to load more rows into a model when the "Load More" button is scrolled into the window.  This creates a scroll and load experience like Salesforce.  There must be enough rows in the model to allow for a scroll though - I'm sure other trigger events could be provided, but this is based on "too many to load at once" in a performance friendly experience, and then load more when scrolling long enough.  We tend to use this with the "Max # of records (Limit)" field set between 50-100, but it will depend on what is being queried and when you see a performance degradation from overloading.  It will require modification of the model and field names to match yours.
```